src/test/java/org/reactome/release/cosmicupdate/COSMICUpdateUtilTest.java

```diff
@@ -0,0 +1,226 @@
+		
+	}
+
+
+	private void createTestCosmicMutantExport() throws IOException
+	{
+		Path pathToMutantExport = Files.createTempFile("mutantExport", "csv");
+		this.COSMICMutantExportFile = pathToMutantExport.toString();
+		Files.writeString(pathToMutantExport, MUTANT_HEADER);
+		Files.writeString(pathToMutantExport, "1234\t1234\tCOSV2323232\n", StandardOpenOption.APPEND);
+		
+	}
+
+
+	private void createTestCosmicFusionExport() throws IOException
+	{
+		Path pathToCOSF = Files.createTempFile("COSF", "csv");
+		this.COSMICFusionExportFile = pathToCOSF.toString();
+		// we only use the FUSION_ID field from the COSMIC Fusion Export.
+		Files.writeString(pathToCOSF, "FUSION_ID\n");
+		Files.writeString(pathToCOSF, "1234\n", StandardOpenOption.APPEND);
+		Files.writeString(pathToCOSF, "COSF6321\n", StandardOpenOption.APPEND);
+		Files.writeString(pathToCOSF, "COSF7890\n", StandardOpenOption.APPEND);
+	}
+	
+	
+	@Test
+	public void testValidateAgainstFiles()
+	{
+		Map<String, COSMICIdentifierUpdater> updates = new HashMap<>();
+		
+		COSMICIdentifierUpdater updater1 = new COSMICIdentifierUpdater();
+		COSMICIdentifierUpdater updater2 = new COSMICIdentifierUpdater();
+		updates.put("1234", updater1);
+		updates.put("COSF1234", updater1);
+		updates.put("5678", updater2);
+		
+		try
+		{
+			COSMICUpdateUtil.validateIdentifiersAgainstFiles(updates, COSMICFusionExportFile, COSMICMutationTrackingFile, COSMICMutantExportFile);
+		}
+		catch (IOException e)
+		{
+			e.printStackTrace();
+			fail();
+		}		
+	}
+	
+	@Test
+	public void testDeterminePrefixes() throws InvalidAttributeException, IOException, Exception
+	{
+		Collection<GKInstance> cosmicObjects = new ArrayList<>();
+		Collection<GKInstance> EWASes = new ArrayList<>();
+		List<GKInstance> mockModResidues = new ArrayList<>();
+		
+		GKInstance mockCosmicObject = Mockito.mock(GKInstance.class);
+		GKInstance mockCosmicObject2 = Mockito.mock(GKInstance.class);
+		GKInstance mockCosmicObject3 = Mockito.mock(GKInstance.class);
+		GKInstance mockEWAS = Mockito.mock(GKInstance.class);
+		GKInstance mockNonEWAS = Mockito.mock(GKInstance.class);
+		GKInstance mockRefSeq = Mockito.mock(GKInstance.class);
+		GKInstance mockRefSeq2 = Mockito.mock(GKInstance.class);
+		GKInstance mockModResidue = Mockito.mock(GKInstance.class);
+		GKInstance mockModResidue2 = Mockito.mock(GKInstance.class);
+		GKInstance mockResiduRefSeq = Mockito.mock(GKInstance.class);
+		GKInstance mockResiduRefSeq2 = Mockito.mock(GKInstance.class);
+		SchemaClass mockEWASSchemaClass = Mockito.mock(SchemaClass.class);
+		SchemaClass mockNonEWASSchemaClass = Mockito.mock(SchemaClass.class);
+		SchemaClass mockFragReplaceSchemaClass = Mockito.mock(SchemaClass.class);
+		SchemaClass mockFragInsertSchemaClass = Mockito.mock(SchemaClass.class);
+		
+		// set up mock object - will not have any EWASes
+		Mockito.when(mockCosmicObject.getAttributeValue(ReactomeJavaConstants.identifier)).thenReturn("5678");
```

Comment:
Could all numeric literals in this test get descriptive constant names for context?